VEDA_OS01/models.py

```diff
@@ -396,6 +396,18 @@ def org(self):
 
         return org
 
+    @property
+    def course_runs(self):
+        course_runs = []
+        if self.local_storedir:
+            course_runs = [
```

Comment:
This is smart. Do you want to do a mini-migration release before this to rename the model field to something more descriptive?

Reply:
@yro this is on our radar, we will do it as a separate ticket, given its usages.